.github/workflows/test_upload_build_to_firebase.yml

```diff
@@ -0,0 +1,78 @@
+name: Test Upload Build to Firebase
+
+# SECRETS needed:
+### SSH_PRIVATE_KEY for Match Repo
+### MATCH_PASS
+### FIREBASE_TOKEN
+### STAGING_FIREBASE_APP_ID
+
+on:
+  pull_request
```

Comment:
As we are uploading app to the firebase based on pull request event, do you think firebase will be overloaded with lots of builds?

Reply:
This is the same as the Test upload build to TestFlight workflow. It is for the test, not for deploying, so it should run to verify the branch before merging.

Reply:
@ducbm051291 I am a bit confused here. As this workflow run the lane `buildStagingAndUploadToFirebase` I think this will upload ipa to the firebase for PR events, please correct me if I am wrong.

Reply:
Yeah, I will upload IPA to the Firebase and the TestFlight. Because this is to verify if our code is deployable 😀